### ISSUE TITLE
[Documentation question] How to make sure all the required PHP extensions are installed

### ISSUE THREAD

**Issue by ByteMorocco** [uilibrary/Stocky-Issues-and-Feature-request#296]
Is there a debug mode or a panel to make sure all the required PHP extensions are properly installed?

I think it should be a self test tab by the application that shows system status and reports server health and dependencies status.

**Comment by hamzakifani** [comment]
Server requirements in Installation  added in latest Release 3.4.0

![server](https://user-images.githubusercontent.com/50219942/127668135-048038e8-98fc-49c2-9771-d634caee3112.PNG)

**Comment by ByteMorocco** [comment]
Awesome

**Comment by ByteMorocco** [comment]
Awesome, but what about the system already installed ? Will it validate the requirements and inform the administrator about current server health ?

**Comment by hamzakifani** [comment]
Great idea . Will add it in setting page ?

**Comment by ByteMorocco** [comment]
@hamzakifani Yes, sure it would be nice if it was in a page called "Site Health" like on WordPress it gives recommendations, highlights critical issues and shows incompatibilities  and missing dependencies.

**Comment by hamzakifani** [comment]
Ok . I will add it in next update . Thanks
